[PR] Added user stories. (#23)
kettle17/group-project-bandcamp: user_stories_README.md

## User Story 1
<!--- "As a X, I want to X, so I can X" -->
"As a musician, I want to see what genres are trending so I can produce music tailored to that genre."

## User Story 2
<!--- "As a X, I want to X, so I can X" -->

> "As a artist manager, I want to be able to see what countries my artist is making the most sales in so I can target those for tours."

Reviewer: Slight grammatical mistake.
```suggestion
"As an artist manager, I want to be able to see which countries my artist is making the most sales in, so I can target those locations for tours."
```

## User Story 3
<!--- "As a X, I want to X, so I can X" -->

> "As a record producer, I want to see what artists are trending the most, so that I can reach out to hire new trending talent."

Reviewer: ```suggestion
"As a record producer, I want to see which artists are trending, so that I can reach out to hire new trending talent."
```

"As a record producer, I want to see which of my artists are selling the most, so that I can change our advertising strategy to promote their music."

## User Story 4
<!--- "As a X, I want to X, so I can X" -->

> "As an avid music listener, I want to see what artists are trending the most so I can discover new music."

Reviewer: Keeping to the template.
```suggestion
"As an avid music listener, I want to see what artists are trending the most, so that I can discover new music."
```

## Developer User Story
<!--- "As a X, I want to X, so I can X" -->

> "As a developer working on the project, I want to understand how the system components/resources interact with each other, so I can build the project architecture."

Reviewer 2: Perhaps adding an acceptance criteria would improve this?
